src/public/view/Reset/Forgot.tsx

```diff
@@ -6,8 +6,10 @@ import ky from 'ky';
 
 import { AuthView } from '../../component/AuthView';
 import { rootRoute } from '../../Root';
-import { EmailSent } from './EmailSent';
+import { useHref } from '../../../util/location';
```

Comment:
Import order. Parent component first and then all the way down to the current working directory.